support/getting-started/engagement-tracking.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Getting Started with Engagement Tracking"
+description: "This guide outlines some of the different ways you can do engagement tracking with SparkPost"
+---
+
+With SparkPost, there are a multitide of ways to configure engagement tracking with our customers.  This guide will outline some of the initial options, and provide references to other guides for configuring custom tracking domains and different ways to use **HTTPS** with your custom tracking domains.
+
+## Getting Started
+Customers can view their engagement through the use of SparkPosts Events.  These events can be searched and viewed in many different ways.  They can be viewed within [SparkPost Signals](https://www.sparkpost.com/docs/reporting/signals-analytics/), queried and searched via the [Events API](https://developers.sparkpost.com/api/events/) or sent to your own environments through the use of [Event Webhooks](https://developers.sparkpost.com/api/webhooks/) to name a few.
+
+
+### Notes on Click Tracking
+SparkPost click tracking links are highly dependent on the size of the original link supplied within the REST Transmission or SMTP request.  If there are concerns over link length within an email with SparkPost's click tracking enabled, first verify the original link length.
+
+> Link sizes of over **4096** bytes will be rejected on redirection
```

Comment:
Do we know approximately how much we add to the original link typically -to give them a hint.  Or is that two difficult to quantify?

Reply:
I guess we could "estimate" just based on some testing, but not sure how much we want to give away.